app/views/shared/_application_information.html.erb

```diff
@@ -17,6 +17,12 @@
             <td class="govuk-table__cell"><strong>Address:</strong></td>
             <td class="govuk-table__cell"><%= @planning_application.site.full_address %></td>
           </tr>
+          <% unless @planning_application.site.postcode.blank? %>
```

Comment:
Seeing as I'm cracking this one open again, `if @planning_application.site.postcode.present?` is a *smidge* more readable, but this is such a minor thing.

Reply:
Now redundant, as we will always have some kind of address (and if we don't the other places where we are calling `.full_address` would give us issues, so I don't think we now need the conditional at all)